### ISSUE TITLE
Jax CKB add gene info

### ISSUE THREAD

**Issue by ahwagner** [ohsu-comp-bio/g2p-aggregator#121]
Jax CKB has normalized gene info available from the source, which we should store/yield when calling `_get_gene_ids()` (https://github.com/ohsu-comp-bio/g2p-aggregator/blob/v0.10/harvester/jax.py#L115-L116) and append to corresponding evidence records.
